docs/whats-new.md

```diff
@@ -2,9 +2,9 @@
 
 Release date: TBD, target Q1-Q2, 2018
 
-### ScreenGridLayer
+### New coloring props for ScreenGridLayer **Experimental**
```

Comment:
Nit: I'd keep the ScreenGridLayer heading, to keep the What's new organized.

The question is how much we want to talk about experimental features here. We generally avoid bringing attention to them but we can make exceptions where it makes sense.

Reply:
I will update the heading,  but I do want to keep the section given enabling these experimental features change final output the layer rendering.